Fatras/include/ActsFatras/EventData/Particle.hpp

```diff
@@ -154,11 +154,15 @@ class Particle {
   constexpr Acts::PdgParticle pdg() const { return m_pdg; }
   /// Particle charge.
   constexpr Scalar charge() const { return m_charge; }
+  /// Particle absolute charge.
+  constexpr Scalar absoluteCharge() const { return std::abs(m_charge); }
   /// Particle mass.
   constexpr Scalar mass() const { return m_mass; }
-  /// Particl qop.
-  constexpr Scalar qop() const {
-    return (charge() == 0 ? 1 : charge()) / absoluteMomentum();
+
+  /// Particl qOverP.
+  constexpr Scalar qOverP() const {
+    return charge() == 0 ? 1 / absoluteMomentum()
+                         : charge() / absoluteMomentum();
```

Comment:
As far as I understand is this change not necessary due to the `unitDirection`->`direction` unification, or am I missing something?

Reply:
right but I would also like to change that so I will adopt the title and description